addons/web/static/src/js/chrome/abstract_action.js

```diff
@@ -105,18 +108,27 @@ var AbstractAction = Widget.extend(ActionMixin, {
         const proms = [this._super(...arguments)];
         if (this.hasControlPanel) {
             if (this.loadControlPanel) {
-                const { context, searchMenuTypes } = this.controlPanelProps;
```

Comment:
context  is never defined in controlPaneProps. Something was missed at some point